frame/bounties/src/lib.rs

```diff
@@ -362,6 +424,12 @@ decl_module! {
 				bounty.status = BountyStatus::CuratorProposed { curator };
 				bounty.fee = fee;
 
+				// Reserve the fee for Curator
+				let bounty_account = Self::bounty_account_id(bounty_id);
```

Comment:
No you can't just reserve other people's fund without their consent. It is reserved when curator accepts the role.

Reply:
Got it Bryan, will resolve it